recipe/meta.yaml

```diff
@@ -7,37 +7,59 @@ package:
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
   sha256: b7c0a9fb8dfcefcfc7ef8fe02a064d194980251d57efaea972cb76005afb3c63
+  patches:
```

Comment:
what is the scope of this ticket ? just add py 3.11 support for existing 1.1.3 version, if so does these patches are needed for older python versions  ?,  it has built previously, is this still can be build for py < 3.11 ? If so maybe those patches should be applied to py 3.11 only ?

Reply:
The package builds for all python versions. Those patches are required for a variety of tests to run succefully. References to PRs that fix those tests have been included in this PR description.
I did not check whether those tests also fail on py != 311, but since those issues that the referenced PRs fix were acknowledged as issues that must be fixed, I think applying these patches to all py version does no harm.